src/main/java/edu/cornell/kfs/fp/batch/service/impl/CreateAccountingDocumentServiceImpl.java

```diff
@@ -81,7 +82,7 @@ protected void processAccountingDocumentFromXml(String fileName) {
             LOG.info("processAccountingDocumentFromXml: Finished processing accounting document XML file: " + fileName);
         } catch (Exception e) {
             reportItem.setXmlSuccessfullyLoaded(false);
-            reportItem.setReportItemMessage(e.getMessage());
+            reportItem.setReportItemMessage(configurationService.getPropertyValueAsString(CuFPKeyConstants.REPORT_CREATE_ACCOUNTING_DOCUMENT_XML_PROCESING_ERROR));

```

Comment:
Is this error message meant to replace the stack trace message, or to be in addition to the stack trace message?

Reply:
The next line will log the exception in the log. I am not sure that the report needs those details. What do you think?

Reply:
That sounds good to me, but it might be a good idea to confirm with the functionals first. Their current JIRA comments mention adding the new message instead of replacing the old message; some additional clarity would be helpful.